src/pages/all-on-one-v2/all-on-one-v2.ts

```diff
@@ -71,6 +76,10 @@ export class AllOnOneV2Page {
     this.slotDetail = this.navParams.get('slotDetail');
   }
 
+  ngAfterViewInit(): void {
```

Comment:
What's the reason you have to do this in a dedicated after initialisation lifecycle hook? Guessing it's to get access to your `ViewChild` to add the fault dynamically - can you confirm?

Reply:
Yes, that is exactly the reason why I can not call the `setUpShowButton` function in the constructor.